manubot/process/tests/test_bibliography.py

```diff
@@ -0,0 +1,41 @@
+import shutil
+
+import pytest
+
+from manubot.pandoc.tests.test_bibliography import (
+    bibliography_paths
+)
+from manubot.process.bibliography import (
+    load_bibliography,
+    load_manual_references,
+)
+
+
+@pytest.mark.skipif(
+    not shutil.which('pandoc-citeproc'),
+    reason='pandoc-citeproc installation not found on system',
+)
+def test_load_multiple_bibliography_paths():
+    citation_to_csl_item = load_manual_references(bibliography_paths)
+    print(list(citation_to_csl_item))
+
+    assert 'doi:10.7554/elife.32822' in citation_to_csl_item
+    csl_item_1 = citation_to_csl_item['doi:10.7554/elife.32822']
+    assert csl_item_1['title'].startswith('Sci-Hub')
+    assert 'CSL JSON Item was loaded by Manubot' in csl_item_1['note']
+    assert 'manual_reference_filename: bibliography.json' in csl_item_1['note']
+    assert 'original_id: doi:10.7554/elife.32822' in csl_item_1['note']
+
+    # raw id corresponding to bibliography.bib
+    assert 'raw:noauthor_techblog:_nodate' in citation_to_csl_item
+    csl_item_2 = citation_to_csl_item['raw:noauthor_techblog:_nodate']
+    assert csl_item_2['title'].startswith('TechBlog')
+    assert 'manual_reference_filename: bibliography.bib' in csl_item_2['note']
+    assert 'original_id: noauthor_techblog:_nodate' in csl_item_2['note']
+
+    # id inferred by pandoc-citeproc during bib2json conversion of .nbib file
```

Comment:
When I run:

```
curl --silent --location https://github.com/manubot/manubot/raw/172bcb9b7da7d0f11528c975d762871700472c77/manubot/pandoc/tests/bibliographies/bibliography.ris \
  | pandoc-citeproc --bib2json --format=ris
```

I get:

<details>
<summary>output</summary>

```json
[
  {
    "DOI": "10.7554/eLife.32822",
    "ISSN": "2050-084X",
    "URL": "https://doi.org/10.7554/eLife.32822",
    "author": [
      {
        "family": "Himmelstein",
        "given": "Daniel S."
      },
      {
        "family": "Romero",
        "given": "Ariel Rodriguez"
      },
      {
        "family": "Levernier",
        "given": "Jacob G."
      },
      {
        "family": "Munro",
        "given": "Thomas Anthony"
      },
      {
        "family": "McLaughlin",
        "given": "Stephen Reid"
      },
      {
        "family": "Greshake Tzovaras",
        "given": "Bastian"
      },
      {
        "family": "Greene",
        "given": "Casey S."
      }
    ],
    "container-title": "eLife",
    "id": "Himmelstein2018",
    "issued": {
      "date-parts": [
        [
          2018,
          3,
          1
        ]
      ]
    },
    "title": "Sci-Hub provides access to nearly all scholarly literature",
    "type": "article-journal",
    "volume": "7"
  },
  {
    "URL": "http://blogs.nature.com/naturejobs/2018/02/20/techblog-manubot-powers-a-crowdsourced-deep-learning-review/",
    "id": "ref2",
    "title": "TechBlog: ‘Manubot’ powers a crowdsourced ‘deep-learning’ review : Naturejobs Blog",
    "title-short": "TechBlog: ‘Manubot’ powers a crowdsourced ‘deep-learning’ review",
    "type": "webpage"
  }
]
```
</details>

So in the case of `.ris`, the `id` field is inferred. I am guessing `id` is always inferred because it seems essential to the way pandoc-citeproc references work, but I cannot find any documentation on this.